.gitlab-ci.yml

```diff
@@ -391,6 +391,8 @@ check-polkadot-companion-build:
       artifacts:                   false
   script:
     - ./.maintain/gitlab/check_polkadot_companion_build.sh
+  after_script:
+    - cd polkadot && git rev-parse --abbrev-ref HEAD
```

Comment:
why put it as a after_script? the companion being used is already printed from within the script.
if you want it shown in the end  i would prefer to have this statement there, as this will eventually fail depending on what the script does.

Reply:
> why put it as a after_script? the companion being used is already printed from within the script.

Yes but it is printed in the beginning and it is not easy to read. When I check a build failure I usually start at the end of the logs to see what is the error. But here the first important information that everyone should see when they open this log is what branch has been used for the build before even reading the error messages.

>  if you want it shown in the end i would prefer to have this statement there, as this will eventually fail depending on what the script does.

I wanted to place it at the end of the build, no matter if it fails or not (since false success are also important). Where do you think I should move it?